.github/workflows/maintenance.yml

```diff
@@ -23,6 +23,6 @@ jobs:
         with:
           dirtyLabel: 'PR: out-of-date'
           repoToken: '${{ secrets.GITHUB_TOKEN }}'
-          commentOnDirty: "This pull request has conflicts, please resolve those before we can evaluate the pull request."
+          commentOnDirty: 'This pull request has conflicts, please resolve those before we can evaluate the pull request.'
```

Comment:
No idea where this change come from...

Reply:
It was changed by Olivier a few days ago.
Consider syncing with `next`. 😉

Reply:
Done :ok_hand: